stix2/test/test_external_reference.py

```diff
@@ -34,7 +35,7 @@ def test_external_reference_capec():
     )
 
     assert str(ref) == CAPEC
-    assert repr(ref) == "ExternalReference(external_id='CAPEC-550', source_name='capec')"
+    assert re.match("ExternalReference\(external_id=u?'CAPEC-550', source_name=u?'capec'\)", repr(ref))
```

Comment:
We don't really need these repr tests. I was just trying to be fancy. It also fails pretty quickly on datetimes.